lnd_services_test.go

```diff
@@ -326,3 +326,29 @@ func TestGetLndInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestCustomMacaroonHex(t *testing.T) {
```

Comment:
Very nice! perhaps just add a godoc to this test describing the functionality we're covering.